pkg/ocicni/ocicni.go

```diff
@@ -585,3 +594,27 @@ func (plugin *cniNetworkPlugin) Status() error {
 	}
 	return nil
 }
+
+// UpdateConfig only support PodCIDR change.
+// No network plugins support changing an already-set PodCIDR
+func (plugin *cniNetworkPlugin) UpdateConfig(name string, content interface{}) {
+	if name != PodCidrUpdateType {
+		return
+	}
+
+	plugin.Lock()
+	defer plugin.Unlock()
+
+	podCidr, ok := content.(string)
+	if !ok {
+		logrus.Warningf("podCIDR update content %v is not string", content)
+		return
+	}
+
+	if plugin.podCidr != "" {
+		logrus.Warningf("PodCIDR has already been set to %s, ignoring update to %s", plugin.podCidr, podCidr)
+		return
+	}
+
+	plugin.podCidr = podCidr
+}
```

Comment:
What happens when the podCidr changes while the container is running?   Do you count on e.g. something like kubenet calling the clearBridgeAddressesExcept() function to blow away the current bridge IP?

Reply:
I think ocicni has no ability to deal with IP conflicts. CRI/postman must make sure the podCidr which need to be passed to ocicni is valid.